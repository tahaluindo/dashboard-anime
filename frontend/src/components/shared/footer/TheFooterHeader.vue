<template>
  <header class="mx-auto mt-10 text-center text-white mb-14 clamptext-11-14">
    {{ title }}
  </header>
</template>

<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({
  props: {
    title: {
      type: String,
      required: true,
    },
  },
});
</script>

<style lang="scss">
</style>