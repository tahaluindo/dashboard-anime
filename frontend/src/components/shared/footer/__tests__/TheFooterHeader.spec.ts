import { mount } from "@vue/test-utils";
import TheFooterHeader from "../TheFooterHeader.vue";

describe("TheFooterHeader component test", () => {
  it("should render the default component", () => {
    const wrapper = mount(TheFooterHeader, {
      props: {
        title: "TITLE",
      },
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
