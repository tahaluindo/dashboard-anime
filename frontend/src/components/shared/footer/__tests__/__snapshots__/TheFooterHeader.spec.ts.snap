// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFooterHeader component test should render the default component 1`] = `"<header class=\\"mx-auto mt-10 text-center text-white mb-14 clamptext-11-14\\">TITLE</header>"`;
